### ISSUE TITLE
fail to load the bin

### ISSUE THREAD

**Issue by YJunkai** [roywang021/UMK#1]
![微信图片_20241113111004](https://github.com/user-attachments/assets/1de72f02-408f-49db-bf7e-fea8f46f9570)
Hello, I would like to ask how to solve this problem?
May I ask how you upload the weight file? 
I wonder if the packet was lost in the upload process, but the file size seems to be no problem

Thank you very much

**Comment by roywang021** [comment]
Did you download the v0 version of vicuna-13b? The size of my "pytorch_model-00001-of-00003.bin" file is 14,025.35 MB. When uploading large files, it’s recommended to use tools that support resumable transfer, e.g., WinSCP.

**Comment by YJunkai** [comment]
Oh, my God, that's my problem.

My local pytorch_model-00001-of-00003.bin is the same size as you said, but the size I uploaded to the server is wrong.

Thank you very much. I will try the WinSCP you suggested for transfer!




Best Regards,

Yang










-----原始邮件-----
发件人:roywang021 ***@***.***>
发送时间:2024-11-13 11:34:08 (星期三)
收件人: roywang021/UMK ***@***.***>
抄送: "Yang Junkai" ***@***.***>, Author ***@***.***>
主题: Re: [roywang021/UMK] fail to load the bin (Issue #1)






Did you download the v0 version of vicuna-13b? The size of my "pytorch_model-00001-of-00003.bin" file is 14,025.35 MB. When uploading large files, it’s recommended to use tools that support resumable transfer, e.g., WinSCP.

**Comment by YJunkai** [comment]
Sorry to bother again, but I also want to ask about the meaning of "After injecting toxic semantics into the adversarial image using the VAJM method "?

This means that I need to run VAJM's 
"python minigpt_visual_attack.py --cfg-path eval_configs/minigpt4_eval.yaml --gpu-id 0 --n_iters 5000 --alpha 1 --save_dir visual_unconstrained "
and after 5000 iterations of generating images, use your 
"python minigpt_vlm_attack.py --cfg-path eval_configs/minigpt4_eval.yaml --gpu-id 0 --n_iters 2000 --alpha 1 --save_dir vlm_unconstrained"
to train another 2000 iterations with 'attack_uncinstrained' in "minigpt_utils/vlm_attacker"?
Because it looks like both images and text in your 'attack_unconstrained' code will continue to train?
If there is something wrong with my understanding, please bear with me. I hope you can point it out to me, thank you very much!

**Comment by roywang021** [comment]
Yes, you're right. Alternatively, you can use the off-the-shelf VAJM example provided in UMK/adversarial_images/prompt_unconstrained_init.bmp, which is copied from the VAJM repo.

**Comment by immortal12389** [comment]
Hello! I downloaded from the Vicuna 13b V0 link provided in the official MiniGPT-4 repository（https://huggingface.co/Vision-CAIR/vicuna/tree/main）. However, the size of the "pytorch_model-00001-of-00003.bin" file I downloaded is 14.7GB. I don't know where the problem lies. I would be really grateful for any guidance.
